client/src/plugins/privacy-preferences/PrivacyPreferences.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import React, { PureComponent } from 'react';
+
+import PrivacyPreferencesView from './PrivacyPreferencesView';
+
+import Flags, { SERVER_INTERACTION } from '../../util/Flags';
+
+const CONFIG_KEY = 'editor.privacyPreferences';
+
+export default class PrivacyPreferences extends PureComponent {
+
+  static IS_PRIVACY_PREFERENCES = true;
+
+  state = {
+    showModal: false,
+    isInitialPreferences: false,
+    preferences: null
+  }
+
+  async componentDidMount() {
+
+    const hasServerInteraction = !!Flags.get(SERVER_INTERACTION) || process.env.NODE_ENV === 'test';
```

Comment:
I added process.env.NODE_ENV here as overwriting Flags.get affects other tests. The other solution would be to overwrite it before the PrivacyPrefernecesSpec and revert it after the test however I'd rather just add this line here than to do that. Open to feedbacks.

Reply:
Ah, that's actually a bad idea as process does not exist in the client side. I thought Atom provided that class.